packages/launchpad/src/migration/OptOutModal.vue

```diff
@@ -0,0 +1,149 @@
+<template>
+  <StandardModal
+    :model-value="true"
+    :title="t('migration.renameAuto.modal.title')"
+    @update:modelValue="emit('cancel')"
+  >
+    <Alert
+      status="warning"
+      :title="t('migration.renameAuto.modal.warning')"
+      :icon="WarningIcon"
+      icon-classes="icon-dark-orange-400"
+    />
+    <MigrationTitle
+      :title="t('migration.renameAuto.title')"
+      class="mt-24px"
+    />
+    <MigrationList>
+      <MigrationListItem>
+        {{ t('migration.renameAuto.modal.line1') }}
+      </MigrationListItem>
+      <MigrationListItem>
+        {{ t('migration.renameAuto.modal.line2') }}
+        <CodeTag class="text-jade-500">
+          [filename].cy.[ext]
+        </CodeTag>
+      </MigrationListItem>
+      <MigrationListItem>
+        {{ t('migration.renameAuto.modal.line3') }}
+      </MigrationListItem>
+      <MigrationListItem>
+        {{ t('migration.renameAuto.modal.line4') }}
+        <CodeTag class="text-jade-500">
+          [filename].cy.[ext]
+        </CodeTag>
+      </MigrationListItem>
+      <MigrationListItem>
+        <i18n-t
+          scope="global"
+          keypath="migration.renameAuto.modal.line5"
+        >
+          <CodeTag class="text-jade-500">
+            [filename].cy.[ext]
+          </CodeTag>
+        </i18n-t>
+      </MigrationListItem>
+      <MigrationListItem>
+        <i18n-t
+          scope="global"
+          keypath="migration.renameAuto.modal.line6"
+        >
+          <CodeTag class="text-jade-500">
+            [filename].cy.[ext]
+          </CodeTag>
+        </i18n-t>
+      </MigrationListItem>
+    </MigrationList>
+    <Radio
+      v-model:value="value"
```

Comment:
There's a lot of `value`s here: Radio's `value`, our `value` ref, the options' `value` attributes. Can we update the ref to be `selectedValue`/`radioValue`/something to help this be a bit more readable?

Reply:
Good call, updated.